### DISCUSSION TITLE
Competition

### DISCUSSION THREAD

**Discussion by rugk** [rugk/unicodify#33] [Ideas]
Just a random discussion about competition and what other tools have/can, what we maybe could add.

Please create issues for new features you'd like to see and link them back here.

**Comment by rugk** [comment]
https://qwerty.dev/fancy-font-generator/ – underline and strikethrough is interesting

**Comment by tdulcet** [comment]
Here are a few more Unicode font conversion websites to get ideas from:

- https://fsymbols.com/all/
- https://coolfont.org/
    - https://coolsymbol.com/cool-fancy-text-generator.html
- https://onlineunicodetools.com/generate-unicode-text
- https://qaz.wtf/u/convert.cgi
- https://lingojam.com/FancyTextGenerator
- https://yaytext.com/styles/
